chroot-bin/croutonclip

```diff
@@ -71,9 +71,9 @@ copyclip() {
             trap "rm -f '$cliptmp'" 0
             cat > $cliptmp
 
-            if ! rundisplay "$next" xclip -o -sel clip \
+            if ! rundisplay "$next" xsel -ob \
                     | diff -q - "$cliptmp" > /dev/null; then
-                cat "$cliptmp" | rundisplay "$next" xclip -i -sel clip
+                rundisplay "$next" xsel -ib < "$cliptmp"
```

Comment:
I don't even know how a useless use of cat snuck into the codebase to begin with.  Thanks for fixing that :)